docs/ingestion/data-formats.md

```diff
@@ -399,7 +399,7 @@ For details, see the Schema Registry [documentation](http://docs.confluent.io/cu
 | type | String | Set value to `schema_registry`. | no |
 | url | String | Specifies the URL endpoint of the Schema Registry. | yes |
 | capacity | Integer | Specifies the max size of the cache (default = Integer.MAX_VALUE). | no |
-| urls | Array<String\> | Specifies the URL endpoints of the multiple Schema Registry instances. | yes (if `url` is not provided) |
+| urls | Array\<String\> | Specifies the URL endpoints of the multiple Schema Registry instances. | yes (if `url` is not provided) |
```

Comment:
undo this change? the first `\` shows up

Reply:
also applies to other instances throughout

Reply:
```suggestion
| urls | Array<String\> | Specifies the URL endpoints of the multiple Schema Registry instances. | yes (if `url` is not provided) |
```